flink-elastic-catalog/src/test/resources/elastic/single-input-event.json

```diff
@@ -1,2 +1,2 @@
 { "create" : { "_index" : "test_single_record_table", "_id" : "1" } }
-{ "float_col": 1.1234, "double_col": 2.123456787, "byte_col": 123, "short_col": 12345, "integer_col": 1, "long_col": 123123123, "unsigned_long_col": 123123123, "half_float_col": 12.123, "scaled_float_col": 12.123, "keyword_col": "flink test", "constant_keyword_col": "flink", "wildcard_col": "flink_*", "binary_col": "U29tZSBiaW5hcnkgYmxvYg==", "date_col": "2015-01-01T12:10:30Z", "date_nanos_col": "2015-01-01T12:10:30.123456789Z", date_epoch_col": 1670249801123, "ip_col": "192.168.1.1", "version_col": "1.2.3", "text_col": "aaa bbb ccc ddd eee ", "boolean_col": true, "text_multifield_col": ["aaa", "bbb", "ccc"]}
```

Comment:
Please fix the file. Missing `"` just before `date_epoch_col`.
Check other files and rows just in case

Reply:
Other files look good :)